### PR TITLE
Add Eventbus tcp options

### PR DESCRIPTION
**Pull request by cescoffier**
These options allow to configure the event bus TCP connections and more especially the TLS.

### CODE REVIEW

File: src/main/asciidoc/java/eventbus.adoc [hunk 1]
```diff
@@ -315,4 +315,65 @@ You can run Vert.x clustered on the command line with
...
 
 If you're registering event bus handlers from inside verticles, those handlers will be automatically unregistered
-when the verticle is undeployed.
\ No newline at end of file
+when the verticle is undeployed.
+
+== Configuring the event bus
+
+The event bus can be configured. It is particularly useful when the event bus is clustered. Under the hood
+the event bus used TCP connections to send and receive message, so the
```
Comment (vietj) [inline src/main/asciidoc/java/eventbus.adoc]: _used_ -> _uses_

File: src/main/java/io/vertx/core/VertxOptions.java [hunk 2]
```diff
@@ -625,6 +605,16 @@ public VertxOptions setWarningExceptionTime(long warningExceptionTime) {
     return this;
   }
 
+  public EventBusOptions getEventBusOptions() {
```
Comment (vietj) [inline src/main/java/io/vertx/core/VertxOptions.java]: missing javadoc

File: src/main/java/io/vertx/core/VertxOptions.java [hunk 3]
```diff
@@ -625,6 +605,16 @@ public VertxOptions setWarningExceptionTime(long warningExceptionTime) {
...
+    return eventBusOptions;
+  }
+
+  public VertxOptions setEventBusOptions(EventBusOptions options) {
```
Comment (vietj) [inline src/main/java/io/vertx/core/VertxOptions.java]: missing javadoc

**Comment by cescoffier** [comment]
@vietj - I've merged the latest master, and marked the JULdelegateTest as ignored for now (still need to find a better way to test this, but this would be in another PR).

**Pull request merged**